.env.development

```diff
@@ -0,0 +1,5 @@
+NEXT_PUBLIC_ENV_NAME=Devlopment
```

Comment:
I used this ligne for testing  purposes (deleted the private testing line , and forgot those since they looked like the other existing one, will remove them in the nex commit